source/utils/moneyFormatter.ts

```diff
@@ -1,10 +1,16 @@
-const formatDuffs = (
-  duffsAmount: number,
-  decimalPlaces = 8,
+const formatDash = (duffsAmount: number, decimalPlaces = 8): number => {
```

Comment:
Please, rename it to `duffsToDash`. Function is not doing formatting anymore.

Also, we don't need `decimalPlaces` argument. Dash is always equal to 10^8 duffs. I guess this argument was added by mistake